frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/VerifyCodeHandler.java

```diff
@@ -339,6 +342,7 @@ private void processSuccessfulCodeRequest(
                     false);
             sessionService.storeOrUpdateSession(
                     session.setVerifiedMfaMethodType(MFAMethodType.SMS));
+            authSessionService.setVerifiedMfaMethodType(authSessionId, MFAMethodType.SMS);
```

Comment:
Wondering what we're going to do about migrating this value from the session table to the auth session table

Reply:
The plan is to set to both the old session and the new session for some period of time (greater than the session expiry). By that point the old and the new should contain the same data, so we can start getting from the new session. We may implement a fallback to get from the old session in case it's not found in the new session. Then we'll stop setting to the old session.